IMDSSample.py

```diff
@@ -1,10 +1,69 @@
 #!/usr/bin/env python3
+
+import base64
 import json
+import subprocess
+
 import requests
 
-endpoint = "http://169.254.169.254/metadata/instance?api-version=2019-04-30"
-headers={'Metadata': 'True'}
+imds_server_base_url = "http://169.254.169.254"
+
+instance_api_version = "2019-03-11"
```

Comment:
We should use 2019-04-30.

Reply:
I avoided using `2019-04-30` since it's not fully rolled out yet, and our MS Docs don't mention that version--I wanted to be consistent :)

Reply:
Check the roll out status and then update MS docs accordingly.

Reply:
Update all the other sample's versions to `2019-03-11`